### PR TITLE
Added check-mk chart to incubator

### PR DESCRIPTION
**Pull request by junaid18183**
Added a check-mk ( monitoring tool on top of Nagios ) chart to incubator

### CODE REVIEW

File: incubator/check-mk/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+engine: gotpl
+home: https://mathias-kettner.de/check_mk.html
+icon: http://git.mathias-kettner.de/git/?p=check_mk.git;a=blob_plain;f=web/htdocs/images/logo_mk.png;h=f41c2227828b79c035fa2e2d3fba5ec939c6f2c3;hb=HEAD
+keywords:
+- check_mk
+- nagios
+- monitoring
+maintainers:
+- email: junaid18183@gmail.com
+  name: Juned Memon
```
Comment (unguiculus) [inline incubator/check-mk/Chart.yaml]: Use Github username.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR.

File: incubator/check-mk/Chart.yaml (hunk 1 above)
Comment (junaid18183) [reply incubator/check-mk/Chart.yaml]: Thanks , updated the name with git username

**Comment by junaid18183** [comment]
have updated the name with git username

**Review by unguiculus — APPROVED** [review]

**Pull request merged**